tests/unit/misc/test_editor.py

```diff
@@ -157,6 +157,25 @@ def test_double_edit(self, editor):
         with pytest.raises(ValueError):
             editor.edit("")
 
+    def test_backup(self, qtbot, message_mock):
+        editor = editormod.ExternalEditor(watch=True)
+        editor.edit('foo')
+        with qtbot.wait_signal(editor.file_updated) as blocker:
+            _update_file(editor._filename, 'bar')
+
+        editor.backup()
+
+        msg = message_mock.getmsg(usertypes.MessageLevel.info)
+        prefix = 'Editor backup at '
+        assert msg.text.startswith(prefix)
+        fname = msg.text[len(prefix):]
+
+        with qtbot.wait_signal(editor.editing_finished) as blocker:
```

Comment:
You don't need the `blocker` (to e.g. read args) here, so don't do `as blocker` :wink: